src/index.html

```diff
@@ -1,12 +1,494 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img
+            src="./images/logo.png"
+            alt="Moyo logo"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__elem is-active" href="#">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__elem" href="#">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__elem" href="#">смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__elem"
+              href="#"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__elem" href="#">гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__elem" href="#">планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__elem" href="#">фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__elem" href="#">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container">
+        <div class="container__element">
+          <div class="card" data-qa="card">
+            <div class="card__picture">
```

Comment:
Do you really need here 4 divs?🤯

Reply:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 1th point